src/Symfony/Component/Routing/Tests/Loader/AnnotationClassLoaderTest.php

```diff
@@ -240,7 +241,7 @@ public function testInvokableClassWithMethodRouteLoad()
 
         $this->assertNull($route, '->load ignores class route');
 
-        $route = $routeCollection->get($methodRouteData['name']);
+        $route = $routeCollection->get($classRouteData['name'].$methodRouteData['name']);
```

Comment:
Technically, that's a BC break, but I don't think this matters much as someone defining a class with an `__invoke` method would probably not define a name on the class `@Route` annotation (I don't see the point).

Reply:
I think you're right, because it would have to map to the `__invoke` method, even if `@Route()` was used for the service definition.